mobile-api/src/index.ts

```diff
@@ -5,31 +5,22 @@ import path from 'path/posix';
 import dbConnect from './db-connect';
 import podcastRoutes from './routes';
 
+Bree.extend(require('@breejs/ts-worker'));
```

Comment:
I'm using the ESM syntax so if I make `@breejs/ts-worker` a DevDeps then I would have to conditionally import the package, which I'm not able to get to work.

Reply:
It's a bit of a mixture at the moment.  'require' is not part of ESM, after all.

It might be worth considering dynamic imports, just so you can keep the syntax consistent, but I wouldn't worry about it for this PR.